india_banking_connector/connectors/doctype/canara_bank_connector/canara_bank_connector.py

```
@@ -0,0 +1,303 @@
+
+		self.AES_KEY = bytes.fromhex(self.get_password("aes_key"))
+
+	@property
+	def urls(self):
+		return super().urls
+
+	@property
+	def headers(self):
+		cert = self.get_file_content(self.cert)
+		cert_key = (
+			cert.lstrip("-----BEGIN CERTIFICATE-----")
```

Ruff (0.15.5)

[warning] Using `.strip()` with multi-character strings is misleading

(B005)